### DISCUSSION TITLE
anyone managed to get vendure deployed on Auzre Functions

### DISCUSSION THREAD

**Discussion by elonmust69** [vendurehq/vendure#1214] [Q&A]
I was thinking and trying to see if it's possible to deploy the vendure on to azure functions.

anyone managed to achieve this or this is just impossible.

**Comment by michaelbromley** [comment]
I've not tried, but it should be possible, since Vendure is in general compatible with serverless architecture. The main challenge will probably be how you deploy the worker, which is usually a continuously-running process which listens for jobs and runs them when they appear. I know others have deployed the worker in a serverless environment by using a custom strategy which wakes up the worker process with an HTTP call when a new job is created.

**Comment by lupeaserban** [comment]
@zhaodadi Did you manage to do it?

I'm curios to what would be the reason you want to deploy Vendure as an Azure function? Does running it in the App Service (Web App) have any downsides?

**Reply by michaelbromley** [reply]
I think the usual argument in favour of deploying as a serverless function is cost savings during idle time.
